### DISCUSSION TITLE
streamlink v 2.3 can't download some low res youtube

### DISCUSSION THREAD

**Discussion by jimdays2788** [streamlink/streamlink#4198] [Q&A]
I have a very limited computer ability, but I am curious about something. I know streamlink is not fast software to download youtube videos, but I would like to know why I can't download some youtube videos in lower resolution. I have Windows 7 and was told streamlink v 2.3 is the latest that will work on this computer, so that is the version I have. For example, this youtube video I can download these resolutions (360p, 240p, 144p,and more):
streamlink "https://www.youtube.com/watch?v=5P9EDJwfXBo" 360p -o output.ts
However, the following youtube video, I can cannot download these resolutions (can download higher resolutions, like 360p)
streamlink "https://www.youtube.com/watch?v=ge4SaeTZ0Mk" 240p -o output.ts
streamlink "https://www.youtube.com/watch?v=ge4SaeTZ0Mk" 144p -o output.ts
Can someone explain why I can't download these resolutions? 
Here is the error message I receive:
Microsoft Windows [Version 6.1.7601]
Copyright (c) 2009 Microsoft Corporation.  All rights reserved.

C:\Users\John>streamlink "https://www.youtube.com/watch?v=ge4SaeTZ0Mk" 240p -o o
utput.ts
[cli][info] Found matching plugin youtube for URL https://www.youtube.com/watch?
v=ge4SaeTZ0Mk
[cli][info] Available streams: audio_mp4a, audio_opus, 144p (worst), 240p, 360p,
 480p, 720p, 720p60, 1080p60 (best)
[cli][info] Opening stream: 240p (muxed-stream)
[cli][error] Try 1/1: Could not open stream <Stream()> (Could not open stream: U
nable to open URL: https://r4---sn-vgqskne6.googlevideo.com/videoplayback?expire
=1637530738&ei=EmiaYdGuBquslu8P1pS3qAI&ip=2600%3A1702%3A1340%3A1820%3A35fa%3Af2d
0%3A8ea8%3Aba13&id=o-AMkO5V5_ekZP1Tl1EpvQ57Azgh9ogIuMi4-PwIKO7NMJ&itag=133&aitag
s=133%2C134%2C135%2C136%2C160%2C242%2C243%2C244%2C247%2C278%2C298%2C299%2C302%2C
303&source=yt_otf&requiressl=yes&mh=Yq&mm=31%2C26&mn=sn-vgqskne6%2Csn-5ualdnel&m
s=au%2Conr&mv=m&mvi=4&pl=42&initcwndbps=1121250&vprv=1&mime=video%2Fmp4&ns=8cKE_
gdIoAmhs49FM9LpUgkG&otf=1&otfp=1&dur=0.000&lmt=1613235091539756&mt=1637508778&fv
ip=6&keepalive=yes&fexp=24001373%2C24007246&c=WEB&n=1YYXec469-ORXfjp&sparams=exp
ire%2Cei%2Cip%2Cid%2Caitags%2Csource%2Crequiressl%2Cvprv%2Cmime%2Cns%2Cotf%2Cotf
p%2Cdur%2Clmt&sig=AOq0QJ8wRgIhAOLF6aBa0WOIzZgCdv26TEvq6DxOdKKfsSjm_dp7rZngAiEAqG
I-sSPaCg1bO7ilSBq-sTzDtnhOtrla7iccpifmWwg%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cm
vi%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRAIgZ6CtWfR5BxqBTYwch-2Yhcj0mOZlX339bm_4Ib1YL
nACIGYYvfp8WQAYLF3pkDCz3WqDNjZmirDpAvzSpV3bhoJJ (404 Client Error: Not Found for
 url: https://r4---sn-vgqskne6.googlevideo.com/videoplayback?expire=1637530738&e
i=EmiaYdGuBquslu8P1pS3qAI&ip=2600%3A1702%3A1340%3A1820%3A35fa%3Af2d0%3A8ea8%3Aba
13&id=o-AMkO5V5_ekZP1Tl1EpvQ57Azgh9ogIuMi4-PwIKO7NMJ&itag=133&aitags=133%2C134%2
C135%2C136%2C160%2C242%2C243%2C244%2C247%2C278%2C298%2C299%2C302%2C303&source=yt
_otf&requiressl=yes&mh=Yq&mm=31%2C26&mn=sn-vgqskne6%2Csn-5ualdnel&ms=au%2Conr&mv
=m&mvi=4&pl=42&initcwndbps=1121250&vprv=1&mime=video%2Fmp4&ns=8cKE_gdIoAmhs49FM9
LpUgkG&otf=1&otfp=1&dur=0.000&lmt=1613235091539756&mt=1637508778&fvip=6&keepaliv
e=yes&fexp=24001373%2C24007246&c=WEB&n=1YYXec469-ORXfjp&sparams=expire%2Cei%2Cip
%2Cid%2Caitags%2Csource%2Crequiressl%2Cvprv%2Cmime%2Cns%2Cotf%2Cotfp%2Cdur%2Clmt
&sig=AOq0QJ8wRgIhAOLF6aBa0WOIzZgCdv26TEvq6DxOdKKfsSjm_dp7rZngAiEAqGI-sSPaCg1bO7i
lSBq-sTzDtnhOtrla7iccpifmWwg%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cini
tcwndbps&lsig=AG3C_xAwRAIgZ6CtWfR5BxqBTYwch-2Yhcj0mOZlX339bm_4Ib1YLnACIGYYvfp8WQ
AYLF3pkDCz3WqDNjZmirDpAvzSpV3bhoJJ))
error: Could not open stream <Stream()>, tried 1 times, exiting

**Comment by mkbloke** [comment]
This happens on the latest version also.  It just seems to happen with some streams, unfortunately.

Ideally, the YT plugin would support their DASH implementation, but until somebody takes the time to do that work, we're stuck with it as it is.

That's not a great answer, but it's the best that can probably be given at the moment.
